README.md

```diff
@@ -163,6 +163,14 @@ configuration passed to the route.
 
 Object with [reply options](https://github.com/fastify/fastify-reply-from#replyfromsource-opts) for `@fastify/reply-from`.
 
+### `disableInternalRewriteLocationHeader`
```

Comment:
Can you use a positive option instead (defaulted to `true`)? Something like `internalRewriteLocationHeader`.

Reply:
Sure, I will do it